### PR TITLE
fix(amplitude): change folder from test app to production app

### LINKED ISSUE #3134: Amplitude: Point External Amplitude to Prod data, not Test data
## User story / feature request

- After the backfill/exports are finished: Point the external_amplitude table to the new folder (304110)

- _As a data analyst/developer, I need Metabase to have data from the Production Benefits app, not the Test app. Currently, Metabase has data from the Test app._

### Acceptance Criteria

- Metabase should now have Production app data ONLY, not mixed in with Test data.
- Use Amplitude to confirm the numbers are accurate.

### Notes

_Please enter any additional information that will facilitate the completion of this ticket. For example: Are there any constraints not mentioned above? Are there any alternatives you have considered?_

### PR DESCRIPTION
**Pull request by machikoyasuda**
# Description

Previously, the Amplitude Data Destination was pulling data from Benefit's _Test environment application_ into Google Cloud Storage, and this _Test environment data_ was being processed via Airflow, and getting cleaned, transformed and ultimately sent to Metabase. But we need Production data, not Test data.

Last week, we updated the Amplitude Data Destination to send data from Benefit's **Production** environment application into Google Cloud Storage, and we started and completed a data backfill job to get all data from August 2022 from Production into this same bucket. The previous Test site data was saved in a folder called `304109/ `, while this new Production data now goes into a new folder called `304110/ `:

image

This pull request changes the Airflow DAG to pull data from this _new Production folder_, `304110`.

## Outstanding questions

@lauriemerrell We had a question - We were wondering if this change will trigger the rest of the resulting flow steps to only display and send data from this new folder `304110`, and delete data from `304109`. What I mean by that is, once this PR is merged, will the resulting Benefits Fact Tables and Metabase charts only show data from `304110`? Or do we have to take some other steps to delete all the old data from `304109`?

Resolves #3134

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation: Added details that this data comes from the Production app.

## How has this been tested?

_Include commands/logs/screenshots as relevant._

## Post-merge follow-ups

_Document any actions that must be taken post-merge to deploy or otherwise implement the changes in this PR (for example, running a full refresh of some incremental model in dbt). If these actions will take more than a few hours after the merge or if they will be completed by someone other than the PR author, please create a dedicated follow-up issue and link it here to track resolution._

- [ ] No action required
- [x] Actions required (specified below): @machikoyasuda to confirm that data in Metabase, GCS are correctly coming from Production app.

### CODE REVIEW

**Comment by lauriemerrell** [comment]
TLDR: This change will update everything downstream and you don't need to do anything else.

Explanation in case people are interested: Every day when the dbt project runs, it re-builds all models (views and tables) from scratch, starting from the configured [sources](https://docs.getdbt.com/docs/build/sources) (in our case, these are the external tables pointing at GCS) and [seeds](https://docs.getdbt.com/docs/build/seeds). The one significant exception in our warehouse is [incremental models](https://docs.calitp.org/data-infra/warehouse/developing_dbt_models.html#incremental-models) where data is appended to the existing table based on lookup rules and re-creating from scratch requires a [full refresh](https://docs.getdbt.com/docs/build/incremental-models#how-do-i-rebuild-an-incremental-model). Because the benefits models are not incremental, you don't need to do anything else to rebuild from scratch.

**Comment by machikoyasuda** [comment]
Marking this PR for review now: @lauriemerrell @thekaveman @angela-tran @allejo

**Review by angela-tran — APPROVED** [review]

**Review by thekaveman — APPROVED** [review]
LGTM!
